Sources/AppCoinsSDK/Domain/Entities/AppcSDKInternal.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AppcSDKInternal.swift
+//
+//
+//  Created by Graciano Caldeira on 11/07/2024.
+//
+
+import Foundation
+
+internal class AppcSDKInternal {
+    
+    static internal func initialize() {
+        AttributionUseCases.shared.getAttribution { result in
+            switch result {
```

Comment:
What is the need for the switch if we are not doing anything with it? Why does the useCase getAttribution return anything on completion if we don't need anything returned?